src/components/Swap/Swap.tsx

```diff
@@ -618,37 +619,31 @@ const Swap: React.FC<{
           </Typography>
         </Box>
       )}
-      {recipient === null && !showWrap && isExpertMode && (
+      {!showWrap && isExpertMode && (
         <Box className={classes.recipientInput}>
           <Box className='header'>
-            <Box />
-            <Button
-              id='add-recipient-button'
-              onClick={() => onChangeRecipient('')}
-            >
-              + Add a send (optional)
-            </Button>
-          </Box>
-        </Box>
-      )}
-      {recipient !== null && !showWrap && (
-        <Box className={classes.recipientInput}>
-          <Box className='header'>
-            <ArrowDown size='16' color='white' />
+            {recipient !== null ? (
+              <ArrowDown size='16' color='white' />
+            ) : (
+              <Box />
+            )}
             <Button
               id='remove-recipient-button'
-              onClick={() => onChangeRecipient(null)}
+              onClick={() => onChangeRecipient(recipient !== null ? null : '')}
```

Comment:
I'm pretty sure we still need to clear the recipient when removing.
e.g if there is a recipient, and then they turn expert mode off will it still send to that recipient?

Reply:
It removes the recipient when toggling off the expert mode